src/components/header/style.module.css

```diff
@@ -327,21 +323,38 @@
 .translation {
 	position: absolute;
 	height: 2rem;
-	top: calc(var(--vh) - 3rem); /* TODO: Why does bottom not work? */
-	display: flex;
+	display: none;
+	bottom: calc(env(safe-area-inset-bottom) + 1rem);
```

Comment:
Hides the icons by default and uses the more straightforward `bottom` now that our parent height has been corrected (see comment above).

What it looks like now:

![Screenshot of header expanded on mobile. The social icons now sit within the viewport at the bottom of the screen](https://github.com/user-attachments/assets/1081c9d0-e30d-4ace-bc50-f6970f0ec28c)